fbw-a380x/src/wasm/systems/a380_systems/src/lib.rs

```diff
@@ -62,6 +67,8 @@ pub struct A380 {
     pressurization_overhead: A380PressurizationOverheadPanel,
     electrical_overhead: A380ElectricalOverheadPanel,
     emergency_electrical_overhead: A380EmergencyElectricalOverheadPanel,
+    payload: A380Payload,
+    airframe: A380Airframe,
```

Comment:
In a future PR this would make sense to refactor, there is also the wing flex module for the A380 that needs to be restructured into ``airframe.rs``, initially this PR wasn't as great in scope which is why this was not done yet.